platform/self_hosting/licensing.mdx

```diff
@@ -0,0 +1,48 @@
+---
+id: licensing
+title: Licensing
+sidebar_label: Licensing
+---
+
+If you use the self-hosted version of Tolgee and want to use the advanced features, you need to subscribe to [one of the paid plans](/pricing/self-hosted).
+You can subscribe to the Business plan directly on the platform.
+
+## Subscribing to a paid plan
+
+First, open the Tolgee platform and go to the [Subscription page of your organization](https://app.tolgee.io/billing-self-hosted).
+
+Then choose the plan you want to subscribe to. If you want, **you can switch to the monthly or yearly billing** cycle (yearly billing is cheaper). After that, click on the "Subscribe" button.

```

Comment:
yearly billing is cheaper -> There is a discount for the yearly plan

Reply:
We have to also mention that the user is going to be billed monthly for the usage.